crates/project-model/src/project_json.rs

```diff
@@ -54,9 +54,9 @@ use paths::{AbsPath, AbsPathBuf, Utf8PathBuf};
 use rustc_hash::FxHashMap;
 use serde::{de, Deserialize, Serialize};
 use span::Edition;
+use std::path::PathBuf;
```

Comment:
> The usages of this should be (now that we depend on it) Utf8Pathbuf.

Whoops, I'll switch it over.

> Though now I also wonder, shouldn't all usages of paths in the file here ideally be `AbsPathBuf`?

I think that's a larger change, but one I'd be happy to make later. I believe that relative paths work today (I did write [this code](https://github.com/facebook/buck2/blob/04e1b643492318c4362f249e07c713d088ba092a/integrations/rust-project/src/main.rs#L106-L108), after all...), so it might be a bit surprising to make this change *now*.